.code-samples.meilisearch.yaml

```diff
@@ -18,7 +18,7 @@ delete_an_index_1: |-
   // OR
   $client->index('movies')->delete();
 get_one_document_1: |-
-  $client->index('movies')->getDocument(25684);
+  $client->index('movies')->getDocument(25684, (new IndexesQuery())->setFields(['id', 'title', 'poster', 'release_date']));
```

Comment:
[Same question](https://github.com/meilisearch/meilisearch-php/pull/347/files#r914846618) here

Reply:
Ok! thanks!
I've updated the code sample, let me know if it's correct then 😄